pkg/config/dashboard.go

```diff
@@ -29,6 +29,11 @@ type ChaosDashboardConfig struct {
 	EnableLeaderElection bool   `envconfig:"ENABLE_LEADER_ELECTION"`
 	Database             *DatabaseConfig
 	PersistTTL           *PersistTTLConfig
+	// ClusterScoped means control Chaos Object in cluster level(all namespace),
+	ClusterScoped bool `envconfig:"CLUSTER_SCOPED" default:"true"`
+	// TargetNamespace is the target namespace to injecting chaos.
+	// It only works with ClusterScoped is false;
+	TargetNamespace string `envconfig:"TARGET_NAMESPACE" default:""`
```

Comment:
Should we set the `TARGET_NAMESPACE` to `chaos-testing`? I noticed that the installation script defaults to chaos-testing.

Reply:
It's configured by `controllerManager.targetNamespace` in `helm/chaos-mesh/values.yaml`, [link](https://github.com/chaos-mesh/chaos-mesh/blob/master/helm/chaos-mesh/values.yaml#L43).

We have a bash script that rendering `instasll.sh` from helm files.

Reply:
Understood, thx for the explanation.~